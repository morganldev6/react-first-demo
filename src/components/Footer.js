import React from "react";

const Footer = () => {
  return (
    <footer>
      <div className="wrapper">
        <p>Â© 2019</p>
      </div>
    </footer>
  );
};

export default Footer;
