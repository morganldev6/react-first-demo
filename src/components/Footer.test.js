import React from "react";
import { render, cleanup } from "@testing-library/react";

import Footer from "./Footer";

afterEach(cleanup);

describe("Footer component", () => {
  test("Renders correctly", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
