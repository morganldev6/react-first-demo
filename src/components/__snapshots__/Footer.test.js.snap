// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders correctly 1`] = `
<div>
  <footer>
    <div
      class="wrapper"
    >
      <p>
        Â© 2019
      </p>
    </div>
  </footer>
</div>
`;
